airflow/models/baseoperator.py

```diff
@@ -1048,6 +1061,14 @@ class BaseOperatorLink(metaclass=ABCMeta):
     Abstract base class that defines how we get an operator link.
     """
 
+    operators = None   # type: Optional[List[BaseOperator]]
```

Comment:
It might make the code a bit cleaner if we do this:

```suggestion
    operators = []   # type: List[BaseOperator]
```

Reply:
This was done mainly for mutable argument gotcha. 

https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
Not an argument though.

Reply:
Jeez !! yeah